src/pages/experiments/[experimentId]/data-processing/index.jsx

```diff
@@ -64,7 +64,7 @@ const DataProcessingPage = () => {
     {
       id: 'comptueEmbeddingFilter',
       name: 'Compute embedding',
-      render: () => <EmbeddingPreview experimentId={experimentId} />,
+      render: (expId) => <EmbeddingPreview experimentId={expId} />,
```

Comment:
better call it `experimentId` instead of `expId`, so that it is consistent with the naming that we use everywhere else

Reply:
It cannot be called that, because `experimentId` is shadowed as a prop.

The reason for this way of rendering is that on initial load (when the variable would be created), `experimentId` would be `undefined`. This causes staging to break (as it does right now) because the data would try to be loaded initially for an undefined experiment. The way this can be resolved is to pass the `experimentId` just when the component is about to render.

Reply:
I see, yeah you are right. Good call regarding the error